src/modules/dashboard/staff/components/staff-data-table.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <v-data-table-virtual
+    :headers="staffTableHeaders"
+    :items="staffStore.staffMembers"
+  >
+    <template #item.actions="{ item }">
+      <TableActionButtons
+        :item-id="item.id"
+        @delete="onDeleteClick"
+        @edit="onEditClick"
+      />
+    </template>
+  </v-data-table-virtual>
+</template>
+
+<script setup lang="ts">
+import { useStaffStore } from '../staff.store';
+import { staffTableHeaders } from '../utils/staff-table-headers';
+import TableActionButtons from '../../components/table-action-buttons.vue';
+
+type StaffDataTableEmits = {
+  (e: 'deleteStaff', staffId: number): void;
+  (e: 'editStaff', staffId: number): void;
+};
```

Comment:
zastanawiam się czy prefix/suffix staff jest wymagany w typach akcji i w nazwie wlasciwosci ID, w sumie kontekst (że działamy w obszarze staff data) jest znam znany po nazwie typu... ale to moje przypuszczenia jedynie.